deimos/openssl/evp.d

```diff
@@ -553,6 +553,10 @@ void	EVP_MD_CTX_init(EVP_MD_CTX* ctx);
 int	EVP_MD_CTX_cleanup(EVP_MD_CTX* ctx);
 EVP_MD_CTX* EVP_MD_CTX_create();
 void	EVP_MD_CTX_destroy(EVP_MD_CTX* ctx);
+EVP_MD_CTX* EVP_MD_CTX_new();
+int     EVP_MD_CTX_reset(EVP_MD_CTX* ctx);
+void    EVP_MD_CTX_free(EVP_MD_CTX* ctx);
+void    EVP_MD_CTX_ctrl(EVP_MD_CTX* ctx, int cmd, int p1, const(void) *p2);
```

Comment:
DStyle of pointers: `const(void)*` 

https://dlang.org/dstyle.html#declaration_style